src/components/cropMarks.css

```diff
@@ -3,7 +3,7 @@
     position: relative;
     background: white;
     border: 40px solid white;
-    z-index: -2000;
+    z-index: 1;
```

Comment:
There was no reason for this to be -2000 and it resulted in me not being able to inspect elements. Any reason why this can't be positive? The whole app is contained in this element so the Z index doesn't really do anything.